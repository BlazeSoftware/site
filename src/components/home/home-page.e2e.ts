import { newE2EPage } from '@stencil/core/testing';

describe('home', () => {
  test('snapshot', async () => {
    const page = await newE2EPage();
    await page.setContent('<home-page></home-page');
    const element = await page.find('home-page');

    await page.waitForChanges();

    expect(element.outerHTML).toMatchSnapshot();
  });
});
