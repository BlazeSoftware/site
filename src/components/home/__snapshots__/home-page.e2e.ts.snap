// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home snapshot 1`] = `"<home-page class=\\"hydrated\\"><main><h1 aria-label=\\"Blaze UI\\" class=\\"c-heading c-site-header\\">Blaze <span class=\\"c-site-header__highlight\\">UI</span></h1><p class=\\"o-container o-container--small u-large u-centered\\">Framework-free open source UI toolkit.</p><p class=\\"o-container o-container--small u-centered u-letter-box-large\\">It provides great structure for building websites quickly with a scalable and maintainable foundation.</p><div class=\\"c-homepage-buttons u-large u-letter-box-large u-centered\\"><stencil-route-link class=\\"hydrated\\"><!----><a class=\\"c-button c-button--brand c-button--rounded\\" href=\\"/getting-started/install\\"><span class=\\"c-button__icon-left\\" aria-hidden=\\"\\"><i class=\\"fas fa-graduation-cap\\"></i></span>Documentation</a></stencil-route-link></div><div class=\\"o-container o-container--super u-centered\\"><div class=\\"o-grid o-grid--wrap o-grid--xsmall-full o-grid--small-full o-grid--medium-full\\"><div class=\\"o-grid__cell o-grid__cell--width-33 u-window-box-super\\"><i class=\\"fa-fw fab fa-accessible-icon fa-5x\\"></i><h3 class=\\"c-heading u-xlarge\\">Accessibility</h3><p class=\\"c-paragraph\\"><span class=\\"u-text--loud\\">Accessibile selectors</span> help to educate the importance of enabling everyone to access your site.</p></div><div class=\\"o-grid__cell o-grid__cell--width-33 u-window-box-super\\"><i class=\\"fa-fw fab fa-js-square fa-5x\\"></i><h3 class=\\"c-heading u-xlarge\\">Framework free</h3><p class=\\"c-paragraph\\"><span class=\\"u-text--loud\\">Any framework</span>, or no framework at all. You're not locked in. It's your choice.</p></div><div class=\\"o-grid__cell o-grid__cell--width-33 u-window-box-super\\"><i class=\\"fa-fw fas fa-hand-paper fa-5x\\"></i><h3 class=\\"c-heading u-xlarge\\">Opt-in</h3><p class=\\"c-paragraph\\">Blaze won't automatically take control of any of your design unless you want it to.</p></div><div class=\\"o-grid__cell o-grid__cell--width-33 u-window-box-super\\"><i class=\\"fa-fw fas fa-mobile fa-5x\\"></i><h3 class=\\"c-heading u-xlarge\\">Responsive</h3><p class=\\"c-paragraph\\">All components are developed mobile first and work on any screen size.</p></div><div class=\\"o-grid__cell o-grid__cell--width-33 u-letter-box-super\\"><i class=\\"fa-fw fas fa-cogs fa-5x\\"></i><h3 class=\\"c-heading u-xlarge\\">Custom builds</h3><p class=\\"c-paragraph\\">In 3 easy steps you can have a custom build with easy to use variables and mixins.</p></div><div class=\\"o-grid__cell o-grid__cell--width-33 u-letter-box-super\\"><i class=\\"fa-fw fas fa-code fa-5x\\" style=\\"color: rgb(116, 116, 140);\\"></i><h3 class=\\"c-heading u-xlarge\\">#UseThePlatform</h3><p class=\\"o-paragraph\\">Our components rely solely on native browser features, not a separate library or framework.</p></div><div class=\\"o-grid__cell o-grid__cell--width-100\\"><h2 class=\\"c-heading u-super\\"><blaze-divider class=\\"hydrated\\"><!----><div class=\\"c-divider c-divider--solid\\"><span class=\\"c-divider__content\\">Open Source</span></div></blaze-divider></h2></div><div class=\\"o-grid__cell o-grid__cell--width-33 u-letter-box-super\\"><i class=\\"fa-fw fab fa-github fa-5x\\"></i><h3 class=\\"c-heading u-xlarge\\">Available to everyone</h3><p class=\\"c-paragraph\\">This project is totally free. It's hosted on Github and is accessible by everyone.</p></div><div class=\\"o-grid__cell o-grid__cell--width-33 u-letter-box-super\\"><i class=\\"fa-fw fas fa-gavel fa-5x\\"></i><h3 class=\\"c-heading u-xlarge\\">MIT Licence</h3><p class=\\"c-paragraph\\">You can benefit from the project however you want. In your own way.</p></div><div class=\\"o-grid__cell o-grid__cell--width-33 u-letter-box-super\\"><i class=\\"fa-fw fas fa-code-branch fa-5x\\"></i><h3 class=\\"c-heading u-xlarge\\">Contributing</h3><p class=\\"c-paragraph\\">See something wrong or want some new feature? Raise an issue or pull request. All feedback is welcome.</p></div></div></div></main></home-page>"`;
